**На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. Код содержит три цифры, которые нужно нажать одновременно.**

**Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?**

Так как кнопки нажимаются одновременно, то порядок не важен, и для формулы вероятности общее число исходов будет определяться количеством сочетаний 3-х элементов из 10,

 а количество благоприятных исходов m=1 - замок открылся.

Вводим функцию расчета исходов

In [2]:
from math import factorial
def combinations(n, k):
    return int(factorial(n) / (factorial(k) * factorial(n - k)))

Расчитываем вероятность открытия замка с первой попытки

In [3]:
P=1/combinations(10,3)
print(f'P(открыть с первой попытки) = {round(P,4)}')

P(открыть с первой попытки) = 0.0083


Ответ: вероятность открытия замка с первой попытки составляет P=0.0083 или примерно 0,83%.